utility/block.go

```diff
@@ -53,28 +57,28 @@ func (u *UtilityContext) CreateAndApplyProposalBlock(proposer []byte, maxTransac
 			}
 			break // we've reached our max
 		}
-		txResult, err := u.ApplyTransaction(txIndex, transaction)
+		txResult, err := u.applyTransaction(txIndex, transaction)
```

Comment:
Similar to a previous comment. There's inconsistency here now. I'd expect to read `u.applyTx(txIndex, tx)` here. And the 2inch dusty fan inside my brain is spinning faster :)

Reply:
I get it. I have a slightly bigger fan and it's also spinning. Let's see if they're spinning at the same frequency.

1. I rename all instances of `transaction` to `tx` in the utility module
2. The comments still use the full word for readability purposes
3.  I keep the protobuf named `Transaction` for clarity purposes

Reply:
> Not a big deal of course but historically, we invested time and thought in these kinds of situations

Fwiw, I did invest a lot (15? mins) of time thinking about it yesterday. If the decision doesn't feel "obvious", let's make it so.

**Reasoning for keeping the `type` called `Transaction`**:
- We refer to types in docs & diagrams
- We refer to types in presentations
- Types (and interfaces) are usually the "starting point" when you are trying to understand a codebase
- The type will define the state commitment (the hash), and there fully being explicit & long is helpful

Reasoning for keeping interfaces & vars `tx`:
- We use this keyword A LOT throughout the codebase
- It adds a lot of reading (longer code lines) and increases the cognitive load on the reader (experienced or not)
- I don't think it adds much value when it's repeated so often
- Being able to reference `someTx` in comments and conversations instead of `someTransaction` will add a lot bit by bit over time

Reasoning for keeping `transaction` in the comments:
- This one I'm open to going either way and feel less strongly about
- These comments will be read less often and by less experienced people than experienced devs
- Using full words "tells a story" of what's actually happening

Reply:
Great points. You have my support

Reply:
Captured it here: https://github.com/pokt-network/pocket/discussions/511

Reply:
Also adding this comment:

```
// `Transaction` is used to name the type for clarity & verbosity, but `tx` is used method signatures
// and variable names to be concise. https://github.com/pokt-network/pocket/pull/503
```